DFC.ServiceTaxonomy.Neo4j/Synonyms/src/main/java/analyzer/SynonymHelper.java

```diff
@@ -0,0 +1,61 @@
+import org.apache.lucene.analysis.Analyzer;
+import org.apache.lucene.analysis.core.LowerCaseFilterFactory;
+import org.apache.lucene.analysis.custom.CustomAnalyzer;
+import org.apache.lucene.analysis.standard.StandardFilterFactory;
+import org.apache.lucene.analysis.standard.StandardTokenizerFactory;
+import org.apache.lucene.analysis.synonym.SynonymFilterFactory;
+import java.io.*;
+import org.apache.commons.io.FileUtils;
+import java.util.Scanner;
+
+public class SynonymHelper {
+
+    public SynonymHelper() {
+    }
+
+    public Analyzer buildAnalyzer(String settingName) {
+			String neo4jPath = System.getenv("NEO4J_CONF") + "/neo4j.conf";
+			String fileUrl = "";
+			
+			File file = new File(neo4jPath);
+
+			try {
+				Scanner scanner = new Scanner(file);
+
+							 int lineNum = 0;
```

Comment:
The indentation is a bit funky 😄.

Reply:
Lack of CTRL+K+D in notepad++ for Java :(